frontend/src/components/Options.js

```diff
@@ -1,39 +1,45 @@
-import React from 'react';
-import styled from 'styled-components'
-import { Password, Expiry, Language } from './Inputs'
+import React from "react";
+import styled from "styled-components";
+import { Password, Expiry, Language } from "./Inputs";
 
 const Flex = styled.div`
-    float: right;
-    display: flex;
-    flex-direction: row;
-    transform: translateY(0.2em);
-    
-    & > *:not(:first-child) {
-      margin-left: 2em;
-    }
+  float: right;
+  display: flex;
+  flex-direction: row;
+  transform: translateY(0.2em);
 
-    @media (max-width: 850px) {
-        float: none !important;
-    }
-`
+  & > *:not(:first-child) {
+    margin-left: 2em;
+  }
 
-const OptionsContainer = ({pass, lang, expiry, onPassChange, onLangChange, onExpiryChange}) => {
-    return (
-        <Flex>
-            <Password
-                value={pass}
-                onChange={onPassChange}
-                id="passwordInput" />
-            <Language
-                value={lang}
-                onChange={onLangChange}
-                id="langInput" />
-            <Expiry
-                value={expiry}
-                onChange={onExpiryChange}
-                id="expiryInput" />
-        </Flex>
-    );
-}
+  @media (max-width: 850px) {
+    position: auto;
+    display: inline-flex;
+  }
+  @media (max-width: 650px) {
+    position: relative;
+    float: none !important;
+    width: 100%;
+    display: inline-table;
+    text-align: center;
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/23178940/114588254-52bc2e80-9c3b-11eb-824e-acc9bf811ee2.png)

any way we can fix this weird spacing issue for the language dropdown?

Reply:
I also didn't get, what causes this? I will look into it.

Reply:
its not a CSS fault i guess.

Reply:
ok I found a way to fix it, leaving a review with required changes

Reply:
can you please suggest it, so I can do it in the same only.